src/ADD/EmployeeContract.java

```diff
@@ -0,0 +1,32 @@
+package ADD;
+
+import java.awt.image.ImageConsumer;
+import java.util.Date;
+
+public class EmployeeContract implements InterfaceDocumentInfoI {
+    private int documentNumber;
```

Comment:
We have common fields for different documents and also common methods.
It make sense to move it to abstract class